pretty_gpx/common/utils/utils.py

```diff
@@ -72,4 +72,19 @@ def format_timedelta(total_seconds: float | int) -> str:
             parts.append(f"{seconds:2.0f}")
 
     # Join the parts with commas
-    return ''.join(parts) if parts else '0s'
\ No newline at end of file
+    return ''.join(parts) if parts else '0s'
+
+
+def points_are_close(p1: tuple[float, float], p2: tuple[float, float], eps: float = 1e-5) -> bool:
+    """Get if points are closed with a tolerance epsilon."""
+    return are_close(p1[0], p2[0], eps=eps) and are_close(p1[1], p2[1], eps=eps)
+
+
+def convert_bytes(size_bytes: int) -> str:
+    """Convert bytes to the most appropriate unit (KB, MB, or GB)."""
+    if size_bytes < 1024 ** 2:
+        return f"{size_bytes / 1024:.2f} KB"
+    elif size_bytes < 1024 ** 3:
+        return f"{size_bytes / (1024 ** 2):.2f} MB"
+    else:
+        return f"{size_bytes / (1024 ** 3):.2f} GB"
```

Comment:
kB, MB, GB... are using a multiplicative factor of 1000
whereas KiB, MiB, GiB, .. use 1024
https://fr.wikipedia.org/wiki/Byte
![image](https://github.com/user-attachments/assets/8d6b0680-c4e9-4a73-9aee-a8227f7a0a3f)